metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -103,8 +112,24 @@ specify the following settings under `kubernetes.yml` in the `data` section:
       hosts: ["https://${NODE_NAME}:10250"]
       bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
       ssl.certificate_authorities:
-        - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+        - ${VALID_CA}
 -----
+NOTE: `VALID_CA` can be any CA that is valid so as to reach out the Kubelet API,
+for instance `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`. According to each specific installation
+of Openshift this can be found either in `secrets` or in `configmaps`.
+For instance, in case of using Openshift installer[https://github.com/openshift/installer/blob/master/docs/user/gcp/install.md]
```

Comment:
Nit. Reorganize it a little bit to more explicitly mention where this can be found. It could be nice to explicitly say where this CA can be found depending on the installation method.
```suggestion
According to each specific installation of Openshift this can be found either in `secrets` or in `configmaps`.
In some installations it can be available as part of the service account secret, in `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`.
In case of using Openshift installer[https://github.com/openshift/installer/blob/master/docs/user/gcp/install.md]
```